src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.page {
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__search {
+  box-sizing: border-box;
+  outline: none;
+  width: 100%;
+  height: 100%;
+
+  background-repeat: no-repeat;
+  background-image: url(./images/Search.svg);
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  transition: box-shadow 0.3s ease;
+
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.search-bar__search--large {
+  background-size: 19px 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+  font-size: 16px;
+  height: 70px;
+}
+
+.search-bar__search--small {
+  background-size: 11px 11px;
+  background-position: 13px center;
+  padding-left: 33px;
+  font-size: 14px;
+  height: 42px;
+}
+
+.search-bar__search:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search-bar__search::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar__search:focus {
+  font-weight: 900;
```

Comment:
You have used 'font-weight: 900;' on the focus state of the search bar, but in the @font-face rule you have declared the font-weight for the 'Avenir-Heavy' font as 900. This means that the text will become heavier when the search bar is focused. If this is the intended behavior, it's fine. Otherwise, you might want to adjust this.